### PR TITLE
Fix and test Beam count  and PrivacyIdCount 

### PR DESCRIPTION
**Pull request by Kulankhina**
Make public partitions optional
Delete value_extractor because it isn't used
Clean-up CountParams comments
Test that Count result is sensible

### CODE REVIEW

File: pipeline_dp/aggregate_params.py [hunk 1]
```diff
@@ -129,21 +129,20 @@ class CountParams:
             unit of privacy (e.g., a user) can participate.
         max_contributions_per_partition: Bounds the number of times one unit of
             privacy (e.g. a user) can contribute to a partition.
+        partition_extractor: A function for partition id extraction from an RDD record.
```
Comment (dvadym) [inline pipeline_dp/aggregate_params.py:132]: please RDD record -> record or "collection record"
 everywhere in this file

Explanation: RDD is a Spark term, this class was introduced initially for Spark only and than used for Beam, but the comment stale

**Review by dvadym — COMMENTED** [review]
Thanks it looks great! I left minor comment

File: pipeline_dp/aggregate_params.py (hunk 1 above)
Comment (Kulankhina) [reply pipeline_dp/aggregate_params.py]: Thanks! fixed.

File: pipeline_dp/aggregate_params.py [hunk 2]
```diff
@@ -104,7 +104,7 @@ class SumParams:
             partition.
         public_partitions: A collection of partition keys that will be present in
             the result.
-        partition_extractor: A function for partition id extraction from an RDD record.
+        partition_extractor: A function for partition id extraction from an collection record.
```
Comment (dvadym) [inline pipeline_dp/aggregate_params.py:107]: an collection -> a collection (and in other places)

Comment (Kulankhina) [reply pipeline_dp/aggregate_params.py]: Thanks! Done.

**Review by dvadym — APPROVED** [review]
Thanks!

**Pull request merged**